boskos/reaper/deployment.yaml

```diff
@@ -14,4 +14,4 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: boskos-reaper
-        image: gcr.io/k8s-testimages/reaper:v20170502-1897e866
+        image: gcr.io/k8s-testimages/reaper:v20170710-2501633f
```

Comment:
oh... I made the image, updated the tag, then git commit --amend.. will make a new one.